source/change-streams/change-streams.rst

```diff
@@ -425,7 +427,7 @@ Once a ``ChangeStream`` has encountered a resumable error, it MUST attempt to re
 - If the ``ChangeStream`` has not received any changes, and ``resumeAfter`` is not specified, and the max wire version is >= ``7``:
 
     - The driver MUST execute the known aggregation command.
-    - The driver MUST specify the ``startAtOperationTime`` key set to the original timestamp from when the changestream was first created.
+    - The driver MUST specify the ``startAtOperationTime`` key set to the original timestamp used in the initial aggregation command.
```

Comment:
I think you should add `startAtOperationTime` as a private member (like we do with `resumeToken`) and reference that here.  Then it's clearer we mean whatever startAtOperationTime was either passed in originally or was set earlier.